src/policy_artifacthub.rs

```diff
@@ -375,8 +375,12 @@ fn compose_install(oci_url: String) -> String {
 ```console
 kwctl pull {}
 ```
+Then, generate the policy manifest and tune it to your liking. For example:
+```console
+kwctl scaffold manifest -t ClusterAdmissionPolicy registry://{}
+```
 "#,
-        oci_url
+        oci_url, oci_url
```

Comment:
I would change the way we build this string to make use of the new string interpolation used by latest versions of rust. I think it would make things more readable:

Do something like 

```
kwctl scaffold manifest -t ClusterAdmissionPolicy registry://{oci_url}
```

So that we can drop the line referenced by this comment